frontend/packages/kubevirt-plugin/src/components/virtualization-overview/VirtOverviewStatusCard.tsx

```diff
@@ -0,0 +1,143 @@
+import * as React from 'react';
+import { GalleryItem, Gallery } from '@patternfly/react-core';
+import { Map as ImmutableMap } from 'immutable';
+import { useTranslation } from 'react-i18next';
+import { connect } from 'react-redux';
+import {
+  isDashboardsOverviewHealthSubsystem as isDynamicDashboardsOverviewHealthSubsystem,
+  DashboardsOverviewHealthSubsystem as DynamicDashboardsOverviewHealthSubsystem,
+  useResolvedExtensions,
+  DashboardsOverviewHealthURLSubsystem as DynamicDashboardsOverviewHealthURLSubsystem,
+  isDashboardsOverviewHealthURLSubsystem as isDynamicDashboardsOverviewHealthURLSubsystem,
+  ResolvedExtension,
+  isResolvedDashboardsOverviewHealthURLSubsystem,
+} from '@console/dynamic-plugin-sdk';
+import { URLHealthItem } from '@console/internal/components/dashboard/dashboards-page/cluster-dashboard/health-item';
+import { DashboardAlerts } from '@console/internal/components/dashboard/dashboards-page/cluster-dashboard/status-card';
+import { FirehoseResource } from '@console/internal/components/utils';
+import { useK8sWatchResource } from '@console/internal/components/utils/k8s-watch-hook';
+import {
+  K8sKind,
+  HealthState,
+  referenceForModel,
+  K8sResourceKind,
+} from '@console/internal/module/k8s';
+import { RootState } from '@console/internal/redux';
+import DashboardCard from '@console/shared/src/components/dashboard/dashboard-card/DashboardCard';
+import DashboardCardBody from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardBody';
+import DashboardCardHeader from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardHeader';
+import DashboardCardLink from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardLink';
+import DashboardCardTitle from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardTitle';
+import HealthBody from '@console/shared/src/components/dashboard/status-card/HealthBody';
+import HealthItem from '@console/shared/src/components/dashboard/status-card/HealthItem';
+import { NetworkAddonsConfigModel } from '../../models';
+
+export const NetworkAddonsConfigResource: FirehoseResource = {
```

Comment:
Do we have a fallback for users that can't access the resources ?
( did we plan what cards in the overview will be available for none-admins ? )

Reply:
hmm, maybe not show the overview for none-admins ?